npm-shrinkwrap.json

```diff
@@ -4877,14 +4877,14 @@
       "resolved": "https://registry.npmjs.org/moment-duration-format/-/moment-duration-format-1.3.0.tgz"
     },
     "mountutils": {
-      "version": "1.0.6",
-      "from": "mountutils@1.0.6",
-      "resolved": "https://registry.npmjs.org/mountutils/-/mountutils-1.0.6.tgz",
+      "version": "1.1.0",
+      "from": "mountutils@latest",
```

Comment:
Ahhh, so does that somehow come from the `node_modules` directory, rather than `package.json`?
If so, that might explain some of the weirdness I was seeing with #1379